devops/ecr/main.tf

```diff
@@ -0,0 +1,19 @@
+module "app-dev-ctx" {
```

Comment:
I'd rename this folder to `resources` ... this folder could be used to provision any development related resource, not repositories only.

Reply:
renamed